api/handlers/user.js

```diff
@@ -119,9 +119,35 @@ var UserHandler = {
           res.status(200).send(fullUrl);
         }
       },
+    },
 
+    // Various leading metrics for all users
+    leading: {
+      get: function(req, res) {
+        var high = req.params.order === 'high' ? -1 : 1
 
+        var query;
+        switch(req.params.datatype) {
+          case 'timestamp':
+            query = User.getLeadersByTimeStamp(high);
+            break;
+
+          case 'expertreps':
```

Comment:
I think we need a default case in the event that the datatype is something different. Otherwise, a datatype that was sent over by some malicious user would crash the system. This ensures that in some kind of strange case we would still set the query to some value